go/vt/wrangler/reparent.go

```diff
@@ -363,7 +363,11 @@ func (wr *Wrangler) plannedReparentShardLocked(ctx context.Context, ev *events.R
 	if topoproto.TabletAliasEqual(shardInfo.MasterAlias, masterElectTabletAlias) {
 		return fmt.Errorf("master-elect tablet %v is already the master", topoproto.TabletAliasString(masterElectTabletAlias))
 	}
-	oldMasterTabletInfo, ok := tabletMap[*shardInfo.MasterAlias]
+	var oldMasterTabletInfo *topo.TabletInfo
+	ok = false
```

Comment:
Well, you've said in the past yourself that PlannedReparentShard should work only when old master is available. :)
If we want to allow it to work without old master available we need to do more changes below. But probably when MasterAlias is nil we shouldn't assume that there's actually no writable MySQL, it's probably just something wrong with the topology (as I believe it was when the problem was reported on slack).